basque.py

```diff
@@ -0,0 +1,80 @@
+from nose.tools import set_trace
+import csv
+import datetime
+import re
+
+from core.metadata_layer import (
+    Metadata,
+    IdentifierData,
+    ContributorData,
+    SubjectData,
+)
+from core.classifier import Classifier
+from core.model import (
+    Identifier,
+    Contributor,
+    Edition,
+)
+from core.util import LanguageCodes
+
+class BasqueMetadataExtractor(object):
+
+    """Transfrom the Basque metadata spreadsheet into a list of Metadata objects."""
+
+    @classmethod
+    def parse(cls, file, data_source_name):
+        metadata_records = []
+        reader = csv.DictReader(file)
+
+        for row in reader:
+            publisher = unicode(row.get('Sello Editorial'), 'utf-8')
+            title = unicode(row.get('Title'), 'utf-8')
+
+            identifier = row.get('ISBN')
```

Comment:
I'm not sure why they're labeled ISBN in the spreadsheet. I can change the label but it might come back if we get a new version.